style/icons/all.less

```diff
@@ -56,7 +56,7 @@
 
 // Note: Use tokens instead of strings to be able to do LESS compile-time validation
 .icon-feedback { .x-icon-bg(feedback); }
-.icon-homework { .x-icon-bg(homework); }
+.icon-exercise, .icon-homework, .icon-end { .x-icon-bg(homework); }
```

Comment:
Can you place selectors on separate lines? Maybe this should be done in the JS that way it's easier to grep for when changing.